build.gradle

```diff
@@ -182,6 +182,10 @@ task runTestsWithTag(type: Test) {
         environment "TERRA_DOCKER_IMAGE", terraDockerImage
 
         if (testTag == "unit") {
+            // set which file in src/test/resources/testconfigs to use
+            String terraTestConfigFileName = terraServer.contains("broad") ? "broad.json" : "verily.json"
```

Comment:
Instead of keying off the server file name, I think it would be better to expose this as a separate test parameter (similar to `terraServer` above), with a default of `broad.json` (since `verily.json` doesn't exist in this repo). This would fully decouple server + test config. That way we preserve the ability run tests against any deployment from any repo. In the short term, I think this would break the nightly tests run from this repo against `verily-devel.json`.